src/openqdc/datasets/qm7x.py

```diff
@@ -39,16 +39,17 @@ class QM7X(BaseDataset):
     # Energy in hartree, all zeros by default
     atomic_energies = np.zeros((MAX_ATOMIC_NUMBER,), dtype=np.float32)
 
-    __energy_methods__ = ["pbe-ts", "mbd"]
+    __energy_methods__ = ["pbe+vdw-ts", "mbd"]
 
     energy_target_names = ["ePBE0", "eMBD"]
 
-    __force_methods__ = ["pbe-ts", "vdw"]
+    __force_methods__ = ["pbe+vdw-ts", "mbd"]
```

Comment:
@FNTwin Are you sure the second force is mbd or just vdw forces?

Reply:
My mistake, corrected. It is a semi empirical tight binding with a many body dispersion correlation and a pbe0 for the dft calculation